Description: skip the test of the simple-arm implementation
 The simple-arm implementation will never be used on any supported (ARMv7
 or above) systems, and fails testing on xenial due to some binutils bug.
 So don't fail the build with a test of code that will never be run.
Author: Steve Langasek <steve.langasek@ubuntu.com>
Index: ffmpeg-2.8.1/libavcodec/arm/dct-test.c
===================================================================
--- ffmpeg-2.8.1.orig/libavcodec/arm/dct-test.c
+++ ffmpeg-2.8.1/libavcodec/arm/dct-test.c
@@ -25,7 +25,6 @@
 };
 
 static const struct algo idct_tab_arch[] = {
-    { "SIMPLE-ARM",     ff_simple_idct_arm,     FF_IDCT_PERM_NONE },
     { "INT-ARM",        ff_j_rev_dct_arm,       FF_IDCT_PERM_LIBMPEG2 },
 #if HAVE_ARMV5TE
     { "SIMPLE-ARMV5TE", ff_simple_idct_armv5te, FF_IDCT_PERM_NONE,      AV_CPU_FLAG_ARMV5TE },
